test-workloads/objdump.bzl

```diff
@@ -0,0 +1,41 @@
+load(
+    "@bazel_tools//tools/cpp:toolchain_utils.bzl",
+    "find_cpp_toolchain",
+    "use_cpp_toolchain",
+)
+
+def _objdump_impl(ctx):
```

Comment:
s/objdump/objcopy -- objdump is not the same is objcopy.

Also, this looks nearly identical to `src/objcopy.bzl`. Can we avoid the duplication?

Reply:
Possibly. I'll look into it. Rules are tricky (to me...)

Reply:
I realize now that these are two different rules. But the point about the name and combining them into a single `.bzl` in the root of the repo still stands.

Reply:
fixed